.github/workflows/gito-react-to-comments.yml

```
@@ -66,16 +66,17 @@ jobs:
         LLM_API_TYPE: anthropic
-        MODEL: "claude-sonnet-4-20250514"
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        LLM_API_KEY: ${{ secrets.ANTHROPIC_API_KEY}}
+        MODEL: claude-sonnet-4-20250514
+        JIRA_TOKEN: ${{ secrets.JIRA_TOKEN }}
+        JIRA_URL: ${{ secrets.JIRA_URL }}
+        JIRA_USER: ${{ secrets.JIRA_USER }}
+        LINEAR_API_KEY: ${{ secrets.LINEAR_API_KEY }}
       run: |
-        gito react-to-comment ${{ github.event.comment.id }} --token "$GITHUB_TOKEN"
+        gito react-to-comment ${{ github.event.comment.id }} --token ${{ secrets.GITHUB_TOKEN }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)